src/utils/getNumberOfFileChanges.ts

```diff
@@ -0,0 +1,27 @@
+import * as vscode from "vscode";
+import getFullBlame from "./getFullBlame";
+
+async function getNumberOfFileChanges(gitAPI:any, lineNumber:number) {
+    let blamePromises = await getFullBlame(
+        lineNumber,
+        lineNumber,
+        vscode.window.activeTextEditor?.document.uri.fsPath,
+        gitAPI
+    );
+
+    let blameArray: string[] = [];
+    let blames = await gitAPI?.repositories[0].blame(
+        vscode.window.activeTextEditor?.document.uri.fsPath || "."
+    );
+
+    blames.split("\n").forEach((line: String) => {
+        blameArray.push(line.split(" ")[0]);
```

Comment:
If this happens all do the time do we really need to split?

Reply:
Yes, because the blame result is a single string and we need a clear way to differentiate what a commit hash is. 

Perhaps, I misunderstood your question?

Reply:
yes it's a string in the shape of:
`commitHash date author message`
but isn't it unique?
Why split it? Why not count the lines?

Reply:
This works: `    const numberOfLines = blames.length;`

But we're over counting changes. We need to decide if a repeated commit counts as a change or not (the discussion above). 

Once that's the decided. We can settle this pretty easily.